project/plugins.sbt

```diff
@@ -0,0 +1,4 @@
+addSbtPlugin("com.eed3si9n"              % "sbt-assembly" % "0.14.9")
+addSbtPlugin("com.github.gseitz"         % "sbt-release"  % "1.0.11")
+addSbtPlugin("io.github.davidgregory084" % "sbt-tpolecat" % "0.1.5")
```

Comment:
INFO: What a strange name for a plugin, but I like what it does.

Reply:
yeah, it's great